### PR TITLE
use default icon if image conversion fails

### PR DESCRIPTION
**Pull request by jdoconnor**

### CODE REVIEW

File: lib/icon_finder.rb [hunk 1]
```diff
@@ -17,6 +17,10 @@ def convert_img (icon, size = '512x512') # convert icon to .png format
     system 'convert', '-resize', size, icon, output, exception: true
   end
   return output, size, 'image/png'
+rescue
```
Comment (supechicken) [inline lib/icon_finder.rb:20]: The `rescue` keyword needs to be used with `begin...end` block:
```ruby
def ...
  begin
    ...
  rescue
    ...
  end
end
```

Comment (jdoconnor) [reply lib/icon_finder.rb:20]: rescue can be scoped to the method without an explicit `begin`. https://github.com/ruby/ruby/blob/d92f09a5eea009fa28cd046e9d0eb698e3d94c5c/doc/syntax/exceptions.rdoc

Comment (jdoconnor) [reply lib/icon_finder.rb:20]: but... if using a begin/end explicit block is useful to the style you want to keep for the project, I'll happily add it.

Comment (supechicken) [reply lib/icon_finder.rb:20]: Didn't know that works before, definitely better than `begin...end`

(also sorry for late replying😅)

**Pull request merged**